histomicstk/cli/NucleiDetection/NucleiDetection.py

```diff
@@ -276,9 +267,19 @@ def main(args):
     else:
         process_whole_image = False
 
-    # retrive style and frame
+    # retrive style
     if not args.style or args.style.startswith('{#control'):
         args.style = None
+
+    # initial arguments
+    it_kwargs = {
+        'tile_size': {'width': args.analysis_tile_size},
+        'scale': {'magnification': args.analysis_mag},
+        'tile_overlap': {'x': args.tile_overlap_value, 'y': args.tile_overlap_value, 'edges': True},
```

Comment:
I don't think there is any benefit in using `edges: True`.  You'll end up with more tiles, but not usefully as the additional area will largely get trimmed or duplicated.